vmware_aria_operations_integration_sdk/mp_build.py

```diff
@@ -146,6 +146,9 @@ def get_docker_registry(
     default_registry_value = f"harbor-repo.vmware.com/vmware_aria_operations_integration_sdk_mps/{adapter_kind_key.lower()}"
 
     original_value = docker_registry
+    print(
```

Comment:
Is this intended to be here, or was it just for debugging?

If intended, there is a typo in `regitry` -> `registry`, and should remove 'docker' in favor of 'container'.